rasa/core/actions/forms.py

```diff
@@ -165,16 +163,43 @@ def _entity_mapping_is_unique(
         mapping_as_string = json.dumps(slot_mapping, sort_keys=True)
         return mapping_as_string in self._unique_entity_mappings
 
-    @staticmethod
+    def get_global_not_intents(self, domain: Domain) -> List[Text]:
+        """Returns a list of ignored intents as retrieved from `global_not_intent`.
+
+        Args:
+            domain: The current model domain.
+
+        Returns:
+            The value/s found in `global_not_intent` parameter in the `domain.yml`
+            (under forms).
+        """
+        if "global_not_intent" in domain.__dict__["forms"][self.name()]:
```

Comment:
optional: In my opinion this violates the [Law of Demeter](https://en.wikipedia.org/wiki/Law_of_Demeter). I find it slightly preferable if we ask the domain to give us the `global_not_intent` for the form (e.g. `domain.global_not_intent_for(self.name()`)